### PR TITLE
User aggregatemetric notification ui

### PR DESCRIPTION
**Pull request by mgrauer**
This is actually done, but I'll WIP it until #201, which this is based on, is merged.  I'll rebase this at that point to clearly show that it only includes 2 commits.

### CODE REVIEW

**Comment by mgrauer** [comment]
Rebased, and ready for review.

File: modules/tracker/views/producer/aggregatemetric.phtml [hunk 1]
```diff
@@ -148,4 +135,48 @@ echo '<script type="text/javascript" src="'.$this->moduleWebroot.'/public/js/pro
...
+                    <th>Actions</th>
+                </tr>
+                <tbody>
+                </tbody>
+        </table>
+        <div id="addAggregateMetricSpecAlertUser">
+            Add a user to be alerted<br/>
+            <div class='alertUserSearch'>
+                    <input type='text' id='addAlertUserSearch' value='' autocomplete='off'
+                       autocorrect='off' autocapitalize='off'/>
```
Comment (cpatrick) [inline modules/tracker/views/producer/aggregatemetric.phtml]: +1 good for mobile. ;)

**Comment by cpatrick** [comment]
LGTM (plus one snarky comment).

File: modules/tracker/views/producer/aggregatemetric.phtml (hunk 1 above)
Comment (mgrauer) [inline modules/tracker/views/producer/aggregatemetric.phtml]: I don't get it.  This was copypasta from elsewhere in Midas, btw.

Comment (cpatrick) [inline modules/tracker/views/producer/aggregatemetric.phtml]: I associate these as fields added to `input`s that make things behave better on mobile browsers. That's all.

**Pull request merged**